### PR TITLE
setup the API and replaced ??? with the result from API

### PR DESCRIPTION
**Pull request by iTrip23**

### CODE REVIEW

File: src/components/Balance.js [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+const Balance = ({ total, currencies, setFromGBPto, convertedValue, fromGBPto }) => {
   return (
     <div className="Balance">
       <h2 className="Balance-title">
         Your account balance is
         <span className="Balance-total">£{total}</span>
       </h2>
       <div className="Balance-alt">
-        Your balance is ??? in
-        <select defaultValue={alternateCurrency}>
```
Comment (makanti) [inline src/components/Balance.js:15]: Where is the closing tag for this select? 🤔

File: src/components/Balance.js [hunk 2]
```diff
@@ -1,23 +1,28 @@
-import React, { useState } from "react";
```
Comment (makanti) [inline src/components/Balance.js:1]: 👌

File: src/App.js [hunk 3]
```diff
@@ -1,32 +1,39 @@
-import React, { Component } from "react";
+import React, { Component, useState, useEffect } from "react";
 import Balance from "./components/Balance";
 import CalcPayment from "./components/CalcPayment";
 import Payments from "./components/Payments";
 import currencies from "./data/currencies";
 import "./App.css";
 
-class App extends Component {
```
Comment (makanti) [inline src/App.js:8]: Great, I did replace the other classes but didn't notice this one. 🙌  great work

File: src/components/Balance.js (hunk 1 above)
Comment (iTrip23) [reply src/components/Balance.js:15]: I will fix it now

**Pull request merged**